classes/02class/exercises/c02-network03/feernandobraga/README.md

```diff
@@ -0,0 +1,41 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+
+  - `aws ec2 create-vpc --cidr-block 192.168.0.0/16 --no-amazon-provided-ipv6-cidr-block --instance-tenancy default`
+    - response:
+      -
+      ```
+      {
+        "Vpc": {
+        "CidrBlock": "192.168.0.0/16",
+        "DhcpOptionsId": "dopt-47c7ec20",
+        "State": "pending",
+        "VpcId": "vpc-0c50c0a08b2a7dc7e",
+        "OwnerId": "667211700992",
+        "InstanceTenancy": "default",
+        "Ipv6CidrBlockAssociationSet": [],
+        "CidrBlockAssociationSet": [
+            {
+                "AssociationId": "vpc-cidr-assoc-00f4c6f5372c20da4",
+                "CidrBlock": "192.168.0.0/16",
+                "CidrBlockState": {
+                    "State": "associated"
+                }
+            }
+        ],
+        "IsDefault": false
+        }
+      }
+      ```
+
```

Comment:
This VPC doesn't have a name. Can you add a name to it?

Reply:
brain fart moment here: I've added the name tag but forgot to add it to my answers.
fixed PR submitted